go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/extension/httpforwarder v0.0.0-00010101000000-000000000000
 	github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/hostobserver v0.0.0-00010101000000-000000000000
 	github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/k8sobserver v0.0.0-00010101000000-000000000000
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.24.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.25.0
```

Comment:
Shouldn't we reference version `v0.0.0-00010101000000-000000000000` like all other local dependencies?

Reply:
done